src/web/tools/KeyEventManager.ts

```diff
@@ -0,0 +1,103 @@
+import { AdaptedEvent, EventTypes } from '../interfaces';
+import EventManager from './EventManager';
+import { PointerType } from '../../PointerType';
+import { View } from 'react-native';
+
+export default class KeyEventManager extends EventManager<HTMLElement> {
+  private activationKeys = ['Enter', 'Space', 'NumpadEnter'];
+  private cancelationKeys = ['Tab'];
+
+  private keyDownCallback = (event: KeyboardEvent): void => {
+    console.log(event.code);
+    if (this.cancelationKeys.indexOf(event.code) !== -1) {
```

Comment:
I'm not sure about using `event.code`. As far as I can see `keyCode` is deprecated, so we have to choose between `code` and `key`. The upside of `key` is that it takes keyboard layout into account (in contrast to `code`, as you can read in the [docs](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent)). On the other hand, `code` provides more information (like `AltLeft` instead of just `Alt`).

I'm not sure which one is better, but maybe that's not something that we should worry about right now (cc @j-piasecki)

Reply:
Given that I know relatively nothing about different keyboard layouts, I'd stick to the one that takes it into account.

Reply:
I agree, changing to `key`

Reply:
Changed as of [3cd0886](https://github.com/software-mansion/react-native-gesture-handler/pull/3035/commits/3cd0886fe23294aa50db89949b726db3d1fba487)